TI-reports/2025/2025-Q1-AI-ML-WG.md

```diff
@@ -0,0 +1,62 @@
+# 2024 Q3 AI/ML WG and model signing project
+
+## Overview
+
+We continue our mission to enable security for AI and use AI for security of open source. The community is healthy, although there are multiple similar groups in various other organisations. This means we have to dedicate time to create interlocks between these groups, to share the knowledge and developments, and make sure work is not duplicated. It also means we have to find the niche where this group can bring added value -- we are currently engaging in determining this. At the same time, we continue work on model signing and experiments towards covering more of the ML supply chain.
+
+## General WG updates
+
+### Purpose
+
+We aim for the WG to be the central place to collect all information regarding ML and security, but we are also planning to deliver technical initiatives, such as the model signing project.
+
+### Current Status
+
+The AI/ML working group now has incubating status.
+
+We have updated the list of interlocks with other working groups in the AI security space. We have started some work on a taxonomy of all these groups and what areas we can collaborate on, what updates we can give/receive, what areas we can work on.  This helps in identifiying gaps and prevent duplication and dillution/fracturing of communities.
```

Comment:
Minor: do you have a link to the list of interlocks with other working groups?

Reply:
Yup, https://docs.google.com/spreadsheets/d/1XOzf0LwksHnVeAcgQ7qMAmQAhlHV2iEf4ICvUwOaOfo/edit?gid=0#gid=0

Will add to the doc too.